nc-times/src/App.jsx

```diff
@@ -1,16 +1,24 @@
 import './App.css'
-import NavBar from './assets/NavBar'
-import Home from './assets/Home'
+import NavBar from './components/NavBar'
+import Home from './components/Home'
+import { Routes , Route } from 'react-router-dom'
+import AllArticles from './components/AllArticles'
 function App() {
 
   return (
-    <div>
+    <>
     <NavBar/>
-    <div>
-   <Home/>
+   
   
-    </div>
-    </div>
+    <Routes>
+   <Route path='/' element ={<Home/>}/>
+   <Route path='/AllArticles' element={<AllArticles/>}/>
```

Comment:
There is no need to mirror the name of your component here! While this is not incorrect, I think /articles would be a more user-friendly endpoint. Also: consider that later we'll have an endpoint for a single article - then you would potentially box yourself into the corner of having something like '/AllArticles/5', which doesn't makes sense, as it would be both all articles _and_ a single article at the same time.